Makefile.inc

```
@@ -15,13 +15,16 @@ APPL_DIRS += \
 	$(mkfile_path)modules\
 	$(mkfile_path)modules/API\
 	$(mkfile_path)modules/calculators\
+	$(mkfile_path)modules/motions\
 	$(mkfile_path)modules/CameraCapture
 
 INCLUDES += \
 	-I$(mkfile_path)modules\
 	-I$(mkfile_path)modules/API\
 	-I$(mkfile_path)modules/calculators\
+	-I$(mkfile_path)modules/motions\
 	-I$(mkfile_path)modules/CameraCapture \
+	-I$(mkfile_path)../../raspike-athrill-v850e2m/sdk/common/library/libcpp-spike/include \
```

パス的には、`-I$(mkfile_path)../../common/library/libcpp-spike/include`だと思います。

```suggestion
	-I$(mkfile_path)../../common/library/libcpp-spike/include \
```